src/FileReader.php

```diff
@@ -38,11 +33,13 @@ final class FileReader
      */
     public static function createSingleton()
```

Comment:
What is the reason this methods stays?

Reply:
I was not focusing on having or not a need behind this method. I found a Singleton creator and unified the implementation with other class